packages/access-api/test/helpers/context.js

```diff
@@ -38,9 +38,11 @@ function createBindings(env) {
 }
 
 /**
- * @param {Partial<AccessApiBindings>} env - environment variables to use when configuring access-api. Defaults to process.env.
+ * @param {object} options
+ * @param {Partial<AccessApiBindings>} [options.env] - environment variables to use when configuring access-api. Defaults to process.env.
+ * @param {unknown} [options.globals] - globals passed into miniflare
```

Comment:
It seems like we could call this `options.miniflare` and type it as `Partial<Parameters<typeof Miniflare>[0]>?

Reply:
When I read `globals` I assumed it would involve a `globalThis` hack

Reply:
It's this option https://miniflare.dev/get-started/api#arbitrary-globals